index.html

```diff
@@ -37,7 +37,160 @@ <h1>I'm Newton <br>Glad to see you!</h1>
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-stories1.png" alt="Stories Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Multi-Post Stories
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-tonic2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Tonic
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
```

Comment:
![btn](https://user-images.githubusercontent.com/78251927/205411996-ceaed7be-da87-4f65-830e-2256546c414d.PNG)

- [ ] As per [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please let us implement all button interactions in the styling.

Reply:
Noted sir